Sources/Fuzzilli/Mutators/RuntimeAssistedMutator.swift

```diff
@@ -385,6 +386,8 @@ extension RuntimeAssistedMutator.Action {
             try translateBinaryOperation(.BitOr)
         case .BitwiseXor:
             try translateBinaryOperation(.Xor)
+        case .NCO:
+            try translateBinaryOperation(.NCO)
```

Comment:
You will also need to update the corresponding JavaScript code, see here: https://github.com/googleprojectzero/fuzzilli/blob/d6f77d7b9425c21a0b36b6d2275c21cf5bba1ced/Sources/Fuzzilli/Lifting/JavaScriptRuntimeAssistedMutatorLifting.swift#L391

Reply:
Thanks very much. I missed something here.